packages/server/src/@constants/constants.ts

```diff
@@ -0,0 +1,18 @@
+import { readFileSync } from "fs";
+import { join } from "path";
+import { Admin } from "src/@entities/admin.entity";
+import { User } from "src/@entities/user.entity";
+import configuration from "src/config/configuration";
```

Comment:
```suggestion
import configuration from "src/@config/configuration";
```
`@` 가 빠져서 import가 안되는것 같아여~